src/core/Buttons/SplitButton/SplitButton.tsx

```diff
@@ -11,14 +11,10 @@ import { Placement } from 'tippy.js';
 import SvgCaretDownSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretDownSmall';
 import SvgCaretUpSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretUpSmall';
 
-import { useTheme } from '../../utils';
+import { PolymorphicForwardRefComponent, useTheme } from '../../utils';
 import '@itwin/itwinui-css/css/button.css';
 
-export type SplitButtonProps = {
-  /**
-   * Callback fired on clicking the primary button.
```

Comment:
I think we have `onClick` here for the documentation.

Reply:
I removed it because the type will not be correct with polymorphism